gd-gltg__old/config.js

```diff
@@ -513,6 +513,48 @@ export const gd3 = {
             "legendImage": `${process.env.GEOSERVER_URL}/wms?REQUEST=GetLegendGraphic&VERSION=1.0.0&` +
                 "FORMAT=image/png&WIDTH=20&HEIGHT=20&layer=gltg:impaired-epa"
         },
+        {
+            "layerGroup": "",
+            "title": "State Legislative District - Lower Chamber",
+            "id": "gltg:state-legi-dist-lower",
+            "wms": "https://greatlakestogulf.org/geoserver/wms",
```

Comment:
Please update `wms` and `legendImage` URLs, so they use `process.env.GEOSERVER_URL` instead of hard-coded values. See line 513 for an example.

Reply:
Done. I copied from ilnlrs, I didn't realize gltg was different (we should update ilnlrs too) 😬